lib/src/common/async_extensions.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:async';
```

Comment:
There's a merge issue with this. I've updated w_common to be 2.12 compatible, and it's infrequently changed, so I just updated the head. So if this is merged to head it won't be usable from any current Workiva code. If you merge it behind head, then you'd want to do that as well, and it's effectively forking this package.

Reply:
Yeah, I'm a bit bummed about that, but I don't know if we want to support different branches for old dart versions. FWIW I did post this for review prior to the version upgrade 😝